foreman/start-manual.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Would it be a good idea to use nix-shell here?

Reply:
I think it isn't needed: it'll slow down the startup time and foreman start isn't expected to work at all outside of a nix shell.